docs/DeveloperGuide.md

```diff
@@ -170,15 +170,48 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 This section describes some noteworthy details on how certain features are implemented.
 
 ### 4.1 Company detail panel (UI component)
-
 The `CompanyDetailPanel` allows user to view the company details of the selected company in the company list.
```

Comment:
Grammar. Either "allows users" or "allows the user". Preferably allows the user for consistency with below section.